addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/NestBridgeHandler.java

```diff
@@ -138,10 +142,8 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
     public void dispose() {
         logger.debug("Nest bridge disposed");
         stopAutomaticRefresh();
-        this.accessToken = null;
-        this.lastDataQuery = null;
-        this.pollingJob = null;
-        this.pollingRunnable = null;
+        this.authorizer = null;
+        this.refreshDataJob = null;
     }
```

Comment:
shouldn't redirectUrl be set to null as well?

Reply:
Added.